drivers/gpu/drm/panel/Kconfig

```diff
@@ -385,6 +385,17 @@ config DRM_PANEL_LINCOLNTECH_LCD197
 	  24 bit RGB per pixel. It provides a MIPI DSI interface to
 	  the host.
 
+config DRM_PANEL_BOE_BV045FHM_L00
+	tristate "BOE BV045FHM-L00 panel"
+	depends on OF
+	depends on DRM_MIPI_DSI
+	depends on BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for BV045FHM-L00 panels. 
+	  The panel has a 1080x1620 resolution and uses
+	  24 bit RGB per pixel. It provides a MIPI DSI interface to
+	  the host.
+
```

Comment:
This block will have to be reordered to alphabetically be around other BOE panels.

Same for Makefile (and I assume sdm660_defconfig will be changed too)